tests/integration/targets/test_zabbix_user/tasks/for_zabbix_54_higher.yml

```diff
@@ -10,7 +10,7 @@
     surname: test
     usrgrps:
       - Guests
-    passwd: password
+    passwd: G$jd_79!jw
```

Comment:
Not sure if you wanted to commit this (password)?

Also lets not commit change of `lang: ` parameter or lets unify it at `en_US`. I don't see reason for this change

Reply:
This is just random password in a test and we need to replace default password with something complex as Zabbix 6.0 does not accept 'bad' passwords any longer (unless you change security settings) so to make tests work with Zabbix 6.0 we need to have som 'secure' password.